metricbeat/module/ceph/_meta/docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+== ceph Module
+
+This is the ceph Module.
+
+We used code/reference from:
+https://github.com/influxdata/telegraf/blob/master/plugins/inputs/ceph/ceph.go
```

Comment:
We should make sure to not just copy / paste code from here. Or was this also created by you?